sample/profiles/trace-event/simple-with-samples.json

```diff
@@ -21,16 +21,6 @@
       "args": {
         "name": "mqt_js"
       }
-    },
-    {
```

Comment:
This confirms the fix - even without any importable events the profile still gets imported correctly.